student/src/main/java/tds/student/sql/repository/RemoteExamRepository.java

```diff
@@ -125,11 +136,11 @@ private static boolean isClientError(HttpStatus status) {
     
     try {
       responseEntity = restTemplate.exchange(
-          builder.build().encode().toUri(),
```

Comment:
2 space because that is what they use.  I imagine I didn't have the write code style in my IDE and missed it.